src/validations/rules/ssp.sch

```diff
@@ -1629,6 +1635,17 @@
                 role="error"
                 test="oscal:selected">A FedRAMP SSP information type confidentiality, integrity, or availability impact must the selected
                 impact.</sch:assert>
+            <sch:assert
+                diagnostics="cia-impact-has-adjustment-justification-diagnostic"
+                doc:guide-reference="Guide to OSCAL-based FedRAMP System Security Plans §4.3"
+                id="cia-impact-has-adjustment-justification"
+                role="error"
+                test="
+                    if (oscal:base ne oscal:selected) then
+                        exists(oscal:adjustment-justification)
+                    else
+                        true()">A FedRAMP SSP information type confidentiality, integrity, or availability impact which has differing
+                base and selected impact levels must provide a justification for the difference.</sch:assert>
```

Comment:
On first read, this sentence (and the similar diagnostic message) was a bit hard for me to parse... perhaps it could be reworded a bit.

Reply:
Prose changed in https://github.com/18F/fedramp-automation/pull/276/commits/896ed5fc7801ef0aaf3a1b99e60a99ec7e8e329e.